CONTRIBUTORS.md

```diff
@@ -138,3 +138,4 @@
  - [jomp16](https://github.com/jomp16)
  - [Leon de Klerk](https://github.com/leondeklerk)
  - [CrispyBaguette](https://github.com/CrispyBaguette)
+ - [Chris-Codes-It](https://github.com/Chris-Codes-It)
```

Comment:
This is Emby contributors list. Move to the list above.

Reply:
Done